awx_collection/plugins/modules/instance.py

```diff
@@ -122,6 +137,12 @@ def main():
         new_fields['node_state'] = node_state
     if listener_port is not None:
         new_fields['listener_port'] = listener_port
+    if peers is not None:
+        new_fields['peers'] = peers
+    if peers is None:
```

Comment:
if a user leaves off the option `peers`, does that remove peers?

I think ideally the only way to remove peers is to do something like

```yaml
hostname: hop
node_type: execution
peers:
```

so it is user defined, but just set to empty. If peers is not set, then we should just keep current value.